internal/lib/lib.go

```diff
@@ -481,6 +492,32 @@ func ShutdownApi() {
 	akoApi.ShutDown()
 }
 
+var clusterLabelChecksum uint32
+var clusterKey string
+var clusterValue string
+
+func SetClusterLabelChecksum() {
+	if GetEnableGRBAC() {
+		clusterKey = *GetLabels()[0].Key
```

Comment:
key and values can be set directly or we can make one function call instead of two.